src/pomodoro/containers/PomodoroTitle.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import format from 'date-fns/format'
+import { Helmet } from 'react-helmet'
+import { connect } from 'react-redux'
+import * as fromPomodoro from 'pomodoro/store'
+
+const PomodoroTitle = ({ lapse, title }) => (
+  <Helmet>
+    <title>🕑 {format(lapse, 'mm:ss')} 🗒️ {title}</title>
```

Comment:
Very nice! I think we can just keep the "Tiempo" name on the end. Something like:

* `... | Tiempo`
* `... • Tiempo`
* `... - Tiempo`

Which one do you think is better?

Reply:
Yeah, i think `... - Tiempo` looks better